main.go

```diff
@@ -8,24 +8,10 @@ import (
 	"github.com/tailscale/terraform-provider-tailscale/tailscale"
 )
 
-// version is filled by goreleaser at build time.
-var version = "dev"
```

Comment:
Moved this to `provider.go` as that is the one place it is used currently and to avoid messiness with having to pass the version into the `Provider` method.